2/contrib/jenkins/fix-permissions

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Fix permissions on the given directory to allow group read/write of 
+# regular files and execute of directories.
+find $1 -exec chgrp 0 {} \;
+find $1 -exec chmod g+rw {} \;
+find $1 -type d -exec chmod g+x {} +
```

Comment:
we don't do that in any of our other image repos that share files between versions today, and i'm kind of leery of symlinks 

1) in git repos in general
2) when we start docker add'ing files that are symlinks

i know it's kind of painful (and has been the source of complaints before) but i'd rather keep the same model here until/unless we adopt a wholesale solution.